src/frontend/src/instance/log_handler.rs

```diff
@@ -54,12 +54,11 @@ impl PipelineHandler for Instance {
 
     async fn get_pipeline(
         &self,
-        name: &str,
-        version: PipelineVersion,
+        pipeline_name: &PipelineName,
```

Comment:
I prefer separating the name and version in code. If a combined representation is necessary, consider using the format `schema.pipeline:{version}`.